nix/sources.json

```diff
@@ -5,10 +5,10 @@
         "homepage": null,
         "owner": "input-output-hk",
         "repo": "cardano-byron-proxy",
-        "rev": "9bae138919c31b5aeac8f7244e2b28e5f31d9f4f",
-        "sha256": "0drl4hhmph729g5s2bds0j9sg71jzw3b2cai4cm439dn2xymwc9x",
+        "rev": "824efdae912531d92f29333b482d4d0fbf2f433a",
+        "sha256": "10s3ivzv6hb7mb32imi2dga02vlaibsx8z1m2h7ns8cfkj4yng30",
         "type": "tarball",
-        "url": "https://github.com/input-output-hk/cardano-byron-proxy/archive/9bae138919c31b5aeac8f7244e2b28e5f31d9f4f.tar.gz",
+        "url": "https://github.com/input-output-hk/cardano-byron-proxy/archive/824efdae912531d92f29333b482d4d0fbf2f433a.tar.gz",
```

Comment:
We'll want to bump it to `5538821647728516d5702cbfd9fb3000fcf41082`, the 1.0.5 tag.